src/app/electron/electron-update/electron-update.component.ts

```diff
@@ -7,7 +7,7 @@ import { LoadingService } from 'src/app/shared/loading/loading.service';
 @Component({
   selector: 'app-electron-update',
   templateUrl: './electron-update.component.html',
-  styleUrls: ['./electron-update.component.css'],
+  styleUrls: ['../../../styles/toast.css'],
```

Comment:
In here and the toast component we *do not* want to update the styleUrls. The toast.css file should be a global style sheet like the others. I think it still needs to be loaded in angular.json

Reply:
If that doesn't work, let me know and I can dig in a bit to what's going on